web/src/app/main/components/ToolbarTitle.js

```diff
@@ -102,10 +110,13 @@ function ToolbarTitle() {
 
     return (
       <Typography
-        className={classes.title}
+        className={classes.backPage}
         color='inherit'
         noWrap
-        component='h1'
+        component={AppLink}
+        variant='h6'
+        to='..'
+        replace
```

Comment:
This assumes all titles rendered in the toolbar are clickable so there probably should be a check for which titles are breadcrumbs and which aren't.